### PR TITLE
refactor: change package setup to use vite

### PR DESCRIPTION
**Pull request by Sparkier**
This should make everything work with svelte kit and vite. In a demo svelte kit app I could use it without problems locally. If we want this update, I will make sure to update the sample to use sveltekit.

### CODE REVIEW

**Comment by Sparkier** [comment]
For some reason, `yarn` needs to be run inside the storybook package now before building storybook. I have not had the time to investigate that yet.

**Comment by domoritz** [comment]
@cabreraalex may know something since he's looked into sveltekit before.

**Comment by cabreraalex** [comment]
Okay I updated storybook (got rid of TS because it was breaking and I got tired), and added a sveltekit example, just a copy of the svelte example.

We probably want to double check and test it separately before we send it and update. Or major version update?

File: .eslintrc.js [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
-    "plugin:@typescript-eslint/recommended",
-    "plugin:prettier/recommended",
-  ],
+	parser: '@typescript-eslint/parser',
+	extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended', 'prettier'],
+	plugins: ['svelte3', '@typescript-eslint'],
+	ignorePatterns: ['*.cjs'],
   env: {
-    es6: true,
+		es2017: true,
```
Comment (domoritz) [inline .eslintrc.js:7]:
```suggestion
    es2017: true,
```

File: packages/package.json
Comment (domoritz) [inline packages/package.json:1]: What's this file for?

**Review by domoritz — APPROVED** [review]
Needs some prettification but otherwise looks good. I'm happy to do another major version.

**Comment by Sparkier** [comment]
> Okay I updated storybook (got rid of TS because it was breaking and I got tired), and added a sveltekit example, just a copy of the svelte example.
> 
> We probably want to double check and test it separately before we send it and update. Or major version update?

I also have that implemented I think similarly in TS. Wanted to wait and see if this part is something we want to do before making another PR for the other things. Can post it as an alternative in another PR.

**Comment by domoritz** [comment]
@cabreraalex can you make a final pass and merge + release?

**Pull request merged**